### ISSUE TITLE
Final go-state-types release for NV23

### ISSUE THREAD

**Issue by rjan90** [filecoin-project/go-state-types#292]
Tracking issue for cutting final go-state-types release for NV23, currently in [v0.14.0-rc6](https://github.com/filecoin-project/go-state-types/releases/tag/v0.14.0-rc6).

This will follow the GST release process: https://github.com/filecoin-project/go-state-types?tab=readme-ov-file#release-process. 

Final release depends on:

```[tasklist]
### Tasks
- [ ] https://github.com/filecoin-project/go-state-types/pull/291
```

**Comment by BigLep** [comment]
@rjan90 : should https://github.com/filecoin-project/go-state-types/pull/287 be in the list to include in the final release?

**Comment by rjan90** [comment]
> @rjan90 : should https://github.com/filecoin-project/go-state-types/pull/287 be in the list to include in the final release?

Yes it should be in the final release, and we also got it included as part of the v0.14.0-rc6 tag: https://github.com/filecoin-project/go-state-types/releases/tag/v0.14.0-rc6,
